src/vma/dev/cq_mgr.cpp

```diff
@@ -660,6 +661,7 @@ bool cq_mgr::compensate_qp_poll_success(mem_buf_desc_t* buff_cur)
 		if (m_rx_pool.size() || request_more_buffers()) {
 			do {
 				mem_buf_desc_t *buff_new = m_rx_pool.get_and_pop_front();
+				buff_new->reset_ref_count();
```

Comment:
Previous comment was deleted by GH as function moved to .inl file.
request_more_buffer() transfer some amount of buffers from global pool to work poll m_rx_pool.
After buffer dispatching and processing it will returned to work pool in dirty state. The point of reset_ref_count() here is cleanup when we request buffer from work pool to make sure it is in known initial state.